src/schema/v2/user.ts

```diff
@@ -29,6 +84,12 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
       description: "The given phone number of the user.",
       type: GraphQLString,
     },
+    saleProfileId: {
+      description: "The id of the user's sale profile.",
+      type: GraphQLString,
+      resolve: ({ sale_profile_id }) => sale_profile_id,
+    },
```

Comment:
This field doesn't need to be exposed

Reply:
I think it does because the Forque code needs it in order to send the mutation!

Reply:
You'd just access it through the `saleProfle`

Reply:
I see now - removed with [85cf29f](https://github.com/artsy/metaphysics/pull/3952/commits/85cf29fc45edf19c076a38ad0199cd057b5c5441) thanks for the nudge!!